[2023-11-07T03:23:45.259+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=check_S3_minio_bucket_v01/run_id=manual__2023-11-07T03:23:21.091976+00:00/task_id=2nd_task permission to 509
[2023-11-07T03:23:45.380+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=check_S3_minio_bucket_v01/run_id=manual__2023-11-07T03:23:21.091976+00:00/task_id=2nd_task permission to 509
[2023-11-07T03:23:45.477+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: check_S3_minio_bucket_v01.2nd_task manual__2023-11-07T03:23:21.091976+00:00 [queued]>
[2023-11-07T03:23:45.520+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: check_S3_minio_bucket_v01.2nd_task manual__2023-11-07T03:23:21.091976+00:00 [queued]>
[2023-11-07T03:23:45.523+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-11-07T03:23:45.588+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): 2nd_task> on 2023-11-07 03:23:21.091976+00:00
[2023-11-07T03:23:45.605+0000] {standard_task_runner.py:57} INFO - Started process 29633 to run task
[2023-11-07T03:23:45.611+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'check_S3_minio_bucket_v01', '2nd_task', 'manual__2023-11-07T03:23:21.091976+00:00', '--job-id', '338', '--raw', '--subdir', 'DAGS_FOLDER/check_S3_bucket.py', '--cfg-path', '/tmp/tmph0g5dw0d']
[2023-11-07T03:23:45.620+0000] {standard_task_runner.py:85} INFO - Job 338: Subtask 2nd_task
[2023-11-07T03:23:45.761+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=check_S3_minio_bucket_v01/run_id=manual__2023-11-07T03:23:21.091976+00:00/task_id=2nd_task permission to 509
[2023-11-07T03:23:45.777+0000] {task_command.py:416} INFO - Running <TaskInstance: check_S3_minio_bucket_v01.2nd_task manual__2023-11-07T03:23:21.091976+00:00 [running]> on host c4630ce72677
[2023-11-07T03:23:46.005+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Toema' AIRFLOW_CTX_DAG_ID='check_S3_minio_bucket_v01' AIRFLOW_CTX_TASK_ID='2nd_task' AIRFLOW_CTX_EXECUTION_DATE='2023-11-07T03:23:21.091976+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-07T03:23:21.091976+00:00'
[2023-11-07T03:23:46.011+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-07T03:23:46.021+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install urllib3']
[2023-11-07T03:23:46.066+0000] {subprocess.py:86} INFO - Output:
[2023-11-07T03:23:49.486+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-11-07T03:23:50.061+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3 in /home/***/.local/lib/python3.8/site-packages (1.26.17)
[2023-11-07T03:24:05.284+0000] {subprocess.py:93} INFO - 
[2023-11-07T03:24:05.286+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.3.1
[2023-11-07T03:24:05.287+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-11-07T03:24:05.685+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-11-07T03:24:05.727+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=check_S3_minio_bucket_v01, task_id=2nd_task, execution_date=20231107T032321, start_date=20231107T032345, end_date=20231107T032405
[2023-11-07T03:24:05.786+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-07T03:24:05.814+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
